### ISSUE TITLE
Escaped chars in string produces illegal result

### ISSUE THREAD

**Issue by andelf** [borkdude/rewrite-edn#35]
```clojure
(print (str (borkdude.rewrite-edn/assoc (borkdude.rewrite-edn/parse-string "{}") :a "C+\\")))
;; produces
{:a "C+\"}
;; which is invalid value
```

See-also: https://github.com/logseq/logseq/issues/10218

**Comment by borkdude** [comment]
I think this is due to a rewrite-clj issue:

```
user=> (println (str (rewrite-clj.node/coerce "C+\\")))
"C+\"
```

**Comment by borkdude** [comment]
Bumping rewrite-clj to 1.1.47 seems to fix it.

**Comment by borkdude** [comment]
How are you using rewrite-edn in logseq, in a JVM, CLJS or (n)bb?

**Comment by andelf** [comment]
It's used as CLJS.

**Comment by borkdude** [comment]
Alright, bumping rewrite-clj in your project should fix this then.

**Comment by andelf** [comment]
Many thanks.

**Comment by borkdude** [comment]
Released as 0.4.7
